packages/block-library/src/index.js

```diff
@@ -228,6 +232,12 @@ const getAllBlocks = () => {
 		queryTitle,
 		postAuthorBiography,
 	];
+	if ( window?.__experimentalEnableFormBlocks ) {
```

Comment:
I commented that the registration PHP functions should not be called at all for the dynamic blocks.